src/platforms/dotnet/guides/uwp/index.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: UWP
+description: "Learn more about Sentry's .NET SDK works with Universal Windows platform (UWP) applications."
+redirect_from:
+  - /platforms/dotnet/uwp/
+---
+
+Sentry's .NET SDK works with Universal Windows platform applications through the [Sentry NuGet package](https://www.nuget.org/packages/Sentry). It works with UWP apps running on .NET Native.
+
+## Configuration
+
+In addition to [initializing the SDK with `SentrySdk.Init`](/platforms/dotnet/), you must configure the UWP specific error handler.
+
+<Note>
+
+The SDK should be initialized in the constructor of your application class (usually `App.xaml.cs`). Do not initialize the SDK in the `OnLaunched()` event of the application or the `Hub` will not be initialized correctly.
+
+</Note>
+
+```csharp
+using Sentry;
+using System.Runtime.ExceptionServices;
+using System.Security;
+using Windows.UI.Xaml;
+using UnhandledExceptionEventArgs = Windows.UI.Xaml.UnhandledExceptionEventArgs;
+
+
+sealed partial class App : Application
+{
+    public App()
+    {
+        SentrySdk.Init("___PUBLIC_DSN___");
```

Comment:
```suggestion
        SentrySdk.Init(o => {
        // Tells which project in Sentry to send events to:
        o => o.Dsn = "___PUBLIC_DSN___";
        // When configuring for the first time, to see what the SDK is doing:
        o.Debug = true;
        // Set traces_sample_rate to 1.0 to capture 100% of transactions for performance monitoring.
        // We recommend adjusting this value in production.
        o.TracesSampleRate = 1.0; });
```

Reply:
By default Debug = true will not work on UWP. On the other hand, it works if we use the DebugDiagnosticLogger and replace the Debug.Write by Trace.Write

Reply:
@lucas-zimerman Good point, but lets consider that a bug in the .NET for now. Lets fix the `DebugDiagnosticLogger` as you suggested and ship a new version